api-report/sequence.api.md

```diff
@@ -136,6 +136,8 @@ export interface IIntervalCollection<TInterval extends ISerializableInterval> ex
     gatherIterationResults(results: TInterval[], iteratesForward: boolean, start?: number, end?: number): void;
     // (undocumented)
     getIntervalById(id: string): TInterval | undefined;
+    // (undocumented)
+    getSlideToSegment(lref: LocalReferencePosition): any;
```

Comment:
can we fix the return type here. we shouldn't be using any. it might also help clean up some of the casting